### ISSUE TITLE
Update w3ui website examples

### ISSUE THREAD

**Issue by alanshaw** [web3-storage/w3ui#138]
The code snippets on the website need updating. They are simplified, and mostly just have styles removed, but also have some code simplifications to make it as easy as possible to understand.

refs #117

**Comment by jchris** [comment]
does it make sense to bring those examples into this repo, and then put them into the test suite? eg the website would just pull the files from this repo during build. and the tiny example would be part of the test suite here. (If we decide to do this, we should track it in a new issue)

**Comment by heyjay44** [comment]
@jchris does it make sense to push the piece of work you suggested to the website initiative?
If it could save some time and effort, I think it would be worthwhile.  
cc @dchoi27

**Comment by jchris** [comment]
I think creating the core test suite would be a core engineering thing. Once it is in place, maintaining it can be support
